pkgs/applications/blockchains/trezor-suite/default.nix

```diff
@@ -53,6 +53,10 @@ appimageTools.wrapType2 rec {
   '';
 
   meta = with lib; {
+    # this application does not perform its core functionality.
+    # ref: https://github.com/NixOS/nixpkgs/issues/281975
+    broken = true;
+
```

Comment:
remove empty line:
```suggestion
```

Reply:
for context, i added this in order to provide a visual break between the comment lines and the `broken` attribute, and the rest of the `meta` attributes.  i've removed it in 2beb7de, but personally, i think the original formatting is more readable. let me know your thoughts.